cmd/minikube/cmd/start.go

```diff
@@ -75,8 +75,10 @@ var (
 var startCmd = &cobra.Command{
 	Use:   commandName,
 	Short: "Starts a local OpenShift cluster.",
-	Long: `Starts a local OpenShift cluster using Virtualbox. This command
-assumes you already have Virtualbox installed.`,
+	Long: `Starts a local OpenShift cluster using VirtualBox. You must have VirtualBox installed
```

Comment:
I think this is not quite correct. I need to double check. I think the default varies depends on the OS.